Packs/Legacy/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.yml

```diff
@@ -146,7 +146,7 @@ tags:
 - incidents
 timeout: 300ns
 type: python
-dockerimage: demisto/dateutil
+dockerimage: demisto/python:2.7.17.6981
```

Comment:
Sure we want to do that? I know that usually we don't change the docker image because it can lead to backward compatibility issues, @ronykoz ?

Reply:
From 5.0 we are ok with changing that, but it should be:
```
dockerimage45: demisto/dateutil
dockerimage: demisto/python:2.7.17.6981
```
But why is it python2?
@erezh31 from which version do we have the demisto.dt in the server?